internal/analytics/deferred.go

```diff
@@ -21,6 +21,18 @@ type deferredData struct {
 
 const deferredCfgKey = "deferrer_analytics"
 
+func SetDeferred(da bool) {
+	deferAnalytics = da
+	if deferAnalytics {
+		return
+	}
+	go func() {
+		if err := sendDeferred(sendEvent); err != nil {
+			logging.Errorf("Could not send deferred events: %v", err)
+		}
+	}()
```

Comment:
```suggestion
	eventWaitGroup.Add(1)
	go func() {
		defer eventWaitGroup.Done()
		if err := sendDeferred(sendEvent); err != nil {
			logging.Errorf("Could not send deferred events: %v", err)
		}
	}()
```
To be thorough (take as a nit if you'd like).